### DISCUSSION TITLE
Wiki Contributors Needed!

### DISCUSSION THREAD

**Discussion by aflacc** [aflacc/ProjectFNF#51] [General]
Hey funky gamers!

I feel like the [wiki](https://github.com/aflacc/ProjectFNF/wiki) needs more pages. If anyone here wants to make their own pages in there that explain how to do stuff, please do. If you have your GitHub account linked to your Discord, I will reward you with a special role called "Wiki Contributor"

Please do not ruin the Wiki pages that are already present, and if you think you can improve one please do. I get notifications on who contributes to the wiki so thats how I'm keeping track of who gets the roles

*To connect your Github to your discord, go to settings and select **connected apps***

Some suggestions on what pages you could add:

- [ ] Adding Dialouge
- [ ] Solving common issues
- [ ] Adding Custom Week directories/ Project.xml guide

**Comment by ghost** [comment]
Maybe I could check the wiki for errors? (spelling errors, code errors, grammar errors, etc.)
